src/EditorFeatures/CSharpTest/EditAndContinue/StatementEditingTests.cs

```diff
@@ -9170,6 +9170,178 @@ public void TupleInDelegate()
                 "Update [(int, int) x]@35 -> [(int, int) y]@35");
         }
 
-        #endregion 
+        #endregion
+
+        #region Nullables
```

Comment:
> #region Nullables [](start = 7, length = 18)

These tests are not particularly useful here. What part of the EnC logic are they testing?

They would rather be useful in compilers where we test local slot and lambda mapping - see e.g. ComplexTypes test in LocalSlotMappingTests, MethodWithClosure1 in EditAndContinueClosureTests etc.

Reply:
Removed. Will send a separate PR with compiles tests.

---
In reply to: [301287623](https://github.com/dotnet/roslyn/pull/36770#discussion_r301287623) [](ancestors = 301287623)